app/src/main/java/com/example/palletify/ui/generator/GeneratorUiState.kt

```diff
@@ -9,7 +9,7 @@ import com.example.palletify.data.Palette
 data class GeneratorUiState(
     val numberOfColours: Int = 5,
     val currentPalette: MutableList<Palette.Color> = mutableListOf(),
-    val mode: GenerationMode = GenerationMode.ANY,
+    val mode: GenerationMode = GenerationMode.ANALOGIC,
```

Comment:
is there a reason why we choose analogic? there's logic to change ANY to a random one behind the scenes and we could just leave the option in the dropdown as ANY.

Reply:
I was thinking that if I was using the app, I'd probably want to try each method one at a time to see what they're like instead of leaving it on ANY and letting it randomly choose a generation method. But I can change it back, could make sense to leave it generic in the beginning.